apps/hash-frontend/src/pages/[shortname]/entities/[entity-uuid].page/entity-editor/links-section/link-table/cells/linked-with-cell/entity-selector.tsx

```diff
@@ -87,56 +161,112 @@ export const EntitySelector = ({
     );
   };
 
+  const { uploadFile } = useFileUploads();
+  const { activeWorkspaceAccountId } = useContext(WorkspaceContext);
+
+  const onFileProvided = useCallback(
+    async (file: File) => {
+      if (!activeWorkspaceAccountId) {
+        throw new Error("Cannot upload file without active workspace");
+      }
+
+      // Close the dropdown immediately as we want the file upload to happen in the background
+      onCancel();
```

Comment:
Perhaps we should rename this
```suggestion
      // Close the dropdown immediately as we want the file upload to happen in the background
      onClose();
```

Reply:
Done in https://github.com/hashintel/hash/pull/3104/commits/85bdc36be999274dcf0ba041366d37b00b1df180 

I used the name the calllback was given elsewhere for consistency